translation/locale_en.php

```diff
@@ -276,7 +277,7 @@
 
 /* download.php start */
 $template['download']['title']="Downloading KVIrc";
-$template['download']['text1']="You can obtain KVIrc in three ways:";
+$template['download']['text1']="You can obtain KVIrc in three ways:<br><br><b><u>Note:</b></u> Consider downloading the current nightly Git builds instead of 4.2.0 stable</b>.<br>These current Git builds are quite stable and safe for day to day usage and also are a considerable improvement that contain many bug fixes.<br>For more information <a href=\"https://github.com/kvirc/KVIrc/wiki/Downloading-KVIrc%27s-nightly-source-or-binaries\">click here</a>.";
```

Comment:
would it be better to link that breadcrumb to download page or disable it for now...

This is all temporary until next stable is out anyway.